docs/overview/diagrams.md

```diff
@@ -121,6 +121,44 @@ classDiagram
 
+# AssociativeEither
+
+[![AssociativeCompose hierarchy][AssociativeEither-image]][AssociativeEither-link]
+
+```mermaid
+classDiagram
+  AssociativeEither~F<_>~ <|-- CommutativeEither~F<_>~
+  AssociativeEither~F<_>~ <|-- IdentityEither~F<_>~
+  class AssociativeEither~F<_>~{
+    Either[L, +*]
+    Exit[E, +*]
+    Fiber[E, +*]
+    Schedule[R, E, +*]
+    Try[+*]
+    ZLayer[R, E, +*]
+    ZManaged[R, E, +*]
+    
+    () either[A, B](() => F[A], () => F[B]): F[Either[A, B]]
```

Comment:
Sorry to nitpick, this can be just `=> F[A]` instead of `() => F[A]`, etc...

Reply:
done: 
![image](https://user-images.githubusercontent.com/10689197/99921919-9699b000-2cfb-11eb-9323-4210eb72d2f1.png)